docs/src/main/sphinx/release/release-476.md

```diff
@@ -0,0 +1,195 @@
+# Release 476 (2 Jun 2025)
+
+## General
+
+* Add support for comparing values of `geometry` type. ({issue}`25225`)
+* {{breaking}} Require JDK 24 to run Trino. ({issue}`23498`)
+* Allow configuring `query.max-memory-per-node` and `memory.heap-headroom-per-node` 
+  relative to maximum heap size. ({issue}`25843`)
+* Add feature to deactivate the automated database schema migration for the database 
+  backend for resource groups with the property `resource-groups.db-migrations-enabled`. ({issue)`25451`)
+* Remove the [](/develop/example-http) from the tar.gz archive and the Docker container.  ({issue}`25128`)
+* Fix rare bug when server can hang under load. ({issue}`25816`)
+* Fix regression that prevented graceful shutdown from working. ({issue}`25690`)
+* Fix potential query failure when `fault_tolerant_execution_runtime_adaptive_partitioning_enabled` 
+  is set to `true`. ({issue}`25870`)
+* Fix failure for queries involving casts with `row` types. ({issue}`25864`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Docker image
+
+## CLI
+
+## BigQuery connector
+
+* {{breaking}} Require the `--sun-misc-unsafe-memory-access=allow` JVM configuration 
+  option to run Trino with the connector. ({issue}`25669`)
```

Comment:
Should we say with `the BigQuery connector`. 
I know it is in the `BigQuery connector` section but I would rather make individual release note entries self descriptive without external context.